hack/scale_out_poc/config_haproxy/haproxy.cfg.template

```diff
@@ -0,0 +1,81 @@
+global
```

Comment:
An updated version of this file has already been moved to master branch in #929. This file should be removed.

Reply:
deleted